Apps/Src/Voltage.c

```diff
@@ -162,16 +162,16 @@ SafetyStatus *Voltage_GetModulesInDanger(void){
 		if(i < NUM_BATTERY_MODULES){
 			// Check if battery is in range of voltage limit
 			if(Voltage_GetModuleMillivoltage(i) > MAX_VOLTAGE_LIMIT * MILLI_SCALING_FACTOR || Voltage_GetModuleMillivoltage(i) < MIN_VOLTAGE_LIMIT * MILLI_SCALING_FACTOR) {
```

Comment:
This could be done later but we should separate this so we can specify UNDERVOLTAGE or OVERVOLTAGE dangers